### PR TITLE
Feature/Added SimpleTile and Visibility

### PR DESCRIPTION
**Pull request by randygbrooks**

### CODE REVIEW

File: src/main/game/gameboard/SimpleTile.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+public class SimpleTile{
+    private TerrainEnum Terrain;
+    private ArrayList<Unit> units;
+    private ArrayList<Army> armies;
+    private ArrayList<RallyPoint> rallyPoints;
+    private Structure structure;
+    TileVisibilityEnum visibility = TileVisibilityEnum.INVISIBLE;
+
+    SimpleTile(TerrainEnum tileType, Location location) {
+        Terrain = tileType;
```
Comment (gdscheele) [inline src/main/game/gameboard/SimpleTile.java]: Need to save location

File: src/main/game/gameboard/SimpleTile.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        units = new ArrayList<Unit>();
+        armies = new ArrayList<Army>();
+        rallyPoints = new ArrayList<RallyPoint>();
+        structure = null;
+    }
+
+    public void updateTile(Tile tile) {
+    	if (visibility == TileVisibilityEnum.VISIBLE) {
```
Comment (gdscheele) [inline src/main/game/gameboard/SimpleTile.java]: this isn't doing anything?

File: src/main/game/gameboard/SimpleTile.java (hunk 1 above)
Comment (randygbrooks) [inline src/main/game/gameboard/SimpleTile.java]: Why does it need to save the location?

File: src/main/game/gameboard/SimpleTile.java (hunk 2 above)
Comment (randygbrooks) [inline src/main/game/gameboard/SimpleTile.java]: Not yet no.

File: src/main/game/gameboard/SimpleTile.java (hunk 1 above)
Comment (randygbrooks) [inline src/main/game/gameboard/SimpleTile.java]: Oh I see I accidently left it in the constructor

**Pull request closed without merging**